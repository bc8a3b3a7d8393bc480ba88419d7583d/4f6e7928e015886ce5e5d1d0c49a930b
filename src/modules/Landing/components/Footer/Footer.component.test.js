import { testSnapshots } from '../../../../utils/test.util';

import Component from './Footer.component';

describe('Modules/Landing - Footer', () => {
  testSnapshots(Component, [
    {
      description: 'basic render',
      props: {},
    },
  ]);
});
