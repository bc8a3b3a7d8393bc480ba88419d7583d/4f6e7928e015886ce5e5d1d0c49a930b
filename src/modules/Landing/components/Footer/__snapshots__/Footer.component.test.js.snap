// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modules/Landing - Footer snapshots basic render 1`] = `
<Component
  style={
    Object {
      "backgroundColor": "white",
      "height": 40,
    }
  }
>
  <Button
    containerStyle={Object {}}
    title="landing.footer.continueButton"
    titleStyle={Object {}}
  />
</Component>
`;
